web_accounts_list.json

```diff
@@ -1545,6 +1545,17 @@
          "category" : "dating",
          "valid" : true
       },
+      {
+         "name" : "Kaggle",
+         "check_uri" : "https://www.kaggle.net/{account}",

```

Comment:
Hmm. I'm not able to resolve `kaggle.net`. `https://www.kaggle.com/administrator` works fine for me. Can you confirm the `.net` vs `.com` please?

Reply:
Oh hey you're right. It's `www.kaggle.com`. Not sure how I messed that up.